site/content/en/docs/Advanced/high-availability-agones.md

```diff
@@ -0,0 +1,54 @@
+---
+title: "High Availability Agones"
+date: 2023-02-08T15:02:23Z
+weight: 20
+description: >
+  Learn how to configure your Agones services for high availability.
+---
+
+## High Availability for Agones Controller
+
+When `SplitControllerAndExtensions` is enabled, the `agones-controller` responsibility is split up into `agones-controller`, which enacts the Agones control loop, and `agones-extensions`, which acts as a service endpoint for webhooks and the allocation extension API. Splitting these responsibilities allows the `agones-extensions` pod to be horizontally scaled, making the Agones control plane highly available.
+
+{{< alpha title="`split controller and extensions`" gate="SplitControllerAndExtensions" >}}
+
+{{< alert title="GKE Autopilot Clusters" color="warning" >}}
+`SplitControllerAndExtensions` must be enabled for GKE Autopilot
+{{< /alert >}}
+
+## Extension Pod Configrations 
+
+It is important note is that `helm --set` sets the variable after the yaml inheritance and after helm resolves the chart values. Changing `controller` value through the use of `--set` will not change the values that `extensions` inherited from `controller`. The same applies for when `--values` or `-f` is used. 
+
+To change `controller.numWorkers` to 200 from 100 values and through the use of `helm --set`, add the follow to the `helm` command:
+
+{{< alert color="warning" >}} Important: This will not have any effect on any `extensions` values! {{< /alert >}}
+```
+ ...
+ --set agones.controller.numWorkers=200
+ ...
+```
+
+## Deployment Considerations
+
+When `SplitControllerAndExtensions` is enabled, what was previously a single `agones-controller` pod is deployed as `agones-controller` and 2 `agones-extensions` pods. For example: 
+
+```
+NAME                                 READY   STATUS    RESTARTS   AGE
+agones-allocator-78c6b8c79-h9nqc     1/1     Running   0          23h
+agones-allocator-78c6b8c79-l2bzp     1/1     Running   0          23h
+agones-allocator-78c6b8c79-rw75j     1/1     Running   0          23h
+agones-controller-fbf944f4-vs9xx     1/1     Running   0          23h
+agones-extensions-5648fc7dcf-hm6lk   1/1     Running   0          23h
+agones-extensions-5648fc7dcf-qbc6h   1/1     Running   0          23h
+agones-ping-5b9647874-2rrl6          1/1     Running   0          27h
+agones-ping-5b9647874-rksgg          1/1     Running   0          27h
+```
+
+The number of replicas for `agones-extensions` be set using helm variable [`agones.extensions.replicas`](https://github.com/googleforgames/agones/blob/main/install/helm/agones/values.yaml#L101)
```

Comment:
I think this should link to https://e4313d7-dot-preview-dot-agones-images.appspot.com/site/docs/installation/install-agones/helm/#configuration, not the values.yaml.